test/EventSourcing.Tests/EventSourcing.Tests.csproj

```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{2107B64F-5A2F-4976-80A7-0D37BD080C50}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>EventSourcing.Tests</RootNamespace>
+    <AssemblyName>EventSourcing.Tests</AssemblyName>
```

Comment:
We will need to add this to Test.cmd, as well as create EventSourcing.Tests project in vNext and add it to that Test.cmd as well.
I can do that for you if you prefer, and push it to your branch